samples/bootloader/README.rst

```diff
@@ -85,12 +85,11 @@ The bootloader uses the :ref:`doc_bl_storage` library to access provisioned data
 Requirements
 ************
 
-* One of the following development boards:
+The sample supports the following development kits:
 
-  * |nRF9160DK|
-  * |nRF52840DK|
-  * |nRF52DK|
-  * |nRF51DK|
+.. include:: /includes/boardname_tables/sample_boardnames.txt
+   :start-after: set9_start
+   :end-before: set9_end
```

Comment:
The nRF53 app core is also supported by this sample.

Reply:
Added.